react/features/base/util/uri.ts

```diff
@@ -1,6 +1,13 @@
 import { parseURLParams } from './parseURLParams';
 import { normalizeNFKC } from './strings';
 
+/**
+ * Http status codes.
+ */
+export enum StatusCode {
```

Comment:
maybe rename to STATUS_CODE

Reply:
Fixed the keys to follow the guide https://www.typescriptlang.org/docs/handbook/enums.html